tests/attention/test_cutlass_mla_fp8_output.py

```
@@ -0,0 +1,260 @@
+
+
+def test_cutlass_mla_fp8_non_cutlass_backend_rejected():
+    """o_scale with non-cutlass backend should raise ValueError.
+
+    We directly set _backend to 'fa2' without calling plan() to avoid
+    JIT compilation dependencies. The o_scale check happens before any
+    module call.
+    """
+    device = torch.device("cuda:0")
+
+    q_nope, q_pe, ckv_cache, kpe_cache, kv_lens, page_table = _setup_mla_inputs(
```

Ruff (0.15.12)

[warning] Unpacked variable `kv_lens` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)